tests/fast-integration/test/1-commerce-mock.js

```diff
@@ -61,12 +69,26 @@ function commerceMockTests() {
       await sendEventAndCheckResponse();
     });
 
+    // renew certificate
+    it("CommerceMock should renew it's certificate", async function () {
+      await renewCommerceMockCertificate();
+    });
+    // call lambda and succeed
+    it("order.created.v1 event should trigger the lastorder function", async function () {
+      await sendEventAndCheckResponse();
+    });
+
+    // revoke certificate
+    it("should revoke Commerce Mock certificate", async function () {
```

Comment:
To see if revocated cert is really revocated, we should try to renew it. The renovation should fail in this case

Reply:
When I tried to renew it I got 201 response from Axios, but when we take a look at the test pod log, the error message is included
`Error: Calling Certificate Renewal URL failed with status '403' and body '{\n \"code\": 403,\n \"error\": \"Certificate has been revoked.`

Reply:
Oh, that's bad :c We should enhance the Varkes response for such case then

Reply:
[Here](https://github.com/kyma-incubator/varkes/commit/87199ea1b912a4a69a4b5367d49b59e413fd45b8#) is the response part in Varkes. It looks like it doesn't propagate [the error that occurs while the renewal](https://github.com/kyma-incubator/varkes/blob/master/modules/app-connector/src/server/connection.ts#L193-L194)